include/rospack/rospack.h

```diff
@@ -285,6 +285,12 @@ class ROSPACK_DECL Rosstackage
                   std::string& path);
 
     /**
+     * @brief List license(s) of a stackage whose name is passed.
+     * @param deps Names of the depended stackages.
+     * @param licenses Pairs of (package name, license(s)).
+     */
+    void licenses(std::vector<std::string>& deps, std::set<std::pair<std::string, std::vector<std::string> > >& licenses);
```

Comment:
This will change ABI which needs to be considered when  deciding into which branch to merge this.